bower.json

```diff
@@ -7,17 +7,37 @@
   "license": "http://polymer.github.io/LICENSE.txt",
   "private": true,
   "dependencies": {
-    "platinum-sw": "polymerelements/platinum-sw#^1.3.0",
-    "paper-styles": "polymerelements/paper-styles#^1.0.0",
-    "iron-icons": "polymerelements/iron-icons#^1.0.0",
-    "iron-icon": "polymerelements/iron-icon#^1.0.0",
+    "platinum-sw": "polymerelements/platinum-sw#2.0-preview",
+    "app-storage": "polymerelements/app-storage#2.0-preview",
+    "paper-styles": "polymerelements/paper-styles#2.0-preview",
+    "iron-icons": "polymerelements/iron-icons#2.0-preview",
+    "iron-icon": "polymerelements/iron-icon#2.0-preview",
     "font-roboto": "polymerelements/font-roboto#^1.0.0",
-    "iron-flex-layout": "polymerelements/iron-flex-layout#^1.0.0",
-    "paper-button": "polymerelements/paper-button#^1.0.0",
-    "paper-fab": "polymerelements/paper-fab#^1.0.0",
-    "paper-icon-button": "polymerelements/paper-icon-button#^1.0.0",
+    "iron-flex-layout": "polymerelements/iron-flex-layout#2.0-preview",
+    "paper-button": "polymerelements/paper-button#2.0-preview",
+    "paper-fab": "polymerelements/paper-fab#2.0-preview",
+    "paper-icon-button": "polymerelements/paper-icon-button#2.0-preview",
     "web-animations-js": "web-animations/web-animations-js#^2.2.1",
```

Comment:
When I follow the codelab, I have GET http://localhost:5000/bower_components/web-animations-js/web-animations.min.js 404 errors…

So maybe it was here for a purpose after all? How can I check where the import is triggered from?